README.md

```diff
@@ -1,4 +1,4 @@
-[![mailchimp3 v3.0.13 on PyPi](https://img.shields.io/pypi/v/mailchimp3.svg)](https://pypi.python.org/pypi/mailchimp3)
+[![mailchimp3 v3.0.14 on PyPi](https://img.shields.io/pypi/v/mailchimp3.svg)](https://pypi.python.org/pypi/mailchimp3)
```

Comment:
Don't change this; they'll do that when it's released.

Reply:
Please do change this, specifically in both README files, and the setup file. Additionally, make the changes in a separate commit instead of squashing everything, we don't need pristine history of only single commits and seeing the incremental changes in the history can help someone else learn if they decide to review PRs and comments in a repository like this.